lib/openscap_parser/value_definitions.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'openscap_parser/value_definition'
+require 'active_support/concern'
+
+module OpenscapParser
+  # Methods related to parsing values
+  module ValueDefinitions
+    extend ActiveSupport::Concern
```

Comment:
Please don't do this, activesupport is rails-based and this gem is not even having rails as its dependency.